main.scss

```
@@ -1,14 +1,183 @@
+/*
+ * NOTE:
+ * This file is used to apply our custom theme to Coral
+ * Talk V5 service.
+ *
+ * The stylesheet is built by the Origami Build Service
+ * and the URL for the CSS file is used in Coral Talk
+ * admin panel as the custom CSS URL.
+ *
+ * This is not a normal use case of Origami styles.
+ */
+
+// sass-lint:disable no-qualifying-elements nesting-depth no-important class-name-format
 @import 'src/scss/variables';
 @import 'src/scss/functions';
 @import 'src/scss/mixins';
 
+@import 'o-buttons/main';
+@import 'o-fonts/main';
+@import 'o-colors/main';
+@import 'o-typography/main';
+
 @if ($o-comments-is-silent == false) {
-	@include oComments();
+	@include oComments;
 
 	// Set to silent again to avoid being output twice
 	$o-comments-is-silent: true !global;
 
-	.talk-plugin-infobox-info {
-		background: pink;
+	@include oFontsInclude(MetricWeb, $style: italic);
+	@include oFontsInclude(MetricWeb, $weight: semibold, $style: italic);
+	@include oFontsInclude(MetricWeb, $weight: medium, $style: italic);
+
+	@include oFontsInclude(MetricWeb);
+	@include oFontsInclude(MetricWeb, $weight: semibold);
+	@include oFontsInclude(MetricWeb, $weight: medium);
+
+	@include oFontsInclude(FinancierDisplayWeb, $style: italic);
+	@include oFontsInclude(FinancierDisplayWeb, $weight: semibold, $style: italic);
+	@include oFontsInclude(FinancierDisplayWeb, $weight: medium, $style: italic);
+
+	@include oFontsInclude(FinancierDisplayWeb);
+	@include oFontsInclude(FinancierDisplayWeb, $weight: semibold);
+	@include oFontsInclude(FinancierDisplayWeb, $weight: medium);
+
+	:root {
+		--font-family-sans-serif: 'MetricWeb';
+		--font-family-serif: 'FinancierDisplayWeb';
```

This could be update to use the `o-typography` variable.

```suggestion
		--font-family-sans-serif: $o-typography-serif;
```